src/pretix/base/models/log.py

```diff
@@ -126,7 +127,9 @@ def notification_type(self):
     def organizer(self):
         from .organizer import Organizer
 
-        if self.event:
+        if self.organizer_link:
+            return self.event.organizer_link
```

Comment:
I guess this is a typo?

```suggestion
            return self.organizer_link
```